# 
# Description: Reenables IDE interrupt on Motorola Powerstack II (Utah)
# Patch author: Sven Luther <luther@debian.org>
# Closes: #345424: linux-source-2.6.14: Missing IDE IRQ for PReP Utah (PowerstackII Pro4000)
#
--- linux-2.6-2.6.14+2.6.15-rc7/arch/ppc/platforms/prep_pci.c.orig	2005-12-31 14:51:16.000000000 +0000
+++ linux-2.6-2.6.14+2.6.15-rc7/arch/ppc/platforms/prep_pci.c	2005-12-31 14:58:23.000000000 +0000
@@ -46,7 +46,7 @@
 static char Utah_pci_IRQ_map[23] =
 {
         0,   /* Slot 0  - unused */
-        0,   /* Slot 1  - unused */
+        4,   /* Slot 1  - IDE - SL82C105 */
         5,   /* Slot 2  - SCSI - NCR825A  */
         0,   /* Slot 3  - unused */
         3,   /* Slot 4  - Ethernet - DEC2114x */
